sui-execution/latest/sui-adapter/src/programmable_transactions/execution.rs

```diff
@@ -17,8 +17,10 @@ mod checked {
         programmable_transactions::{
             context::*,
             trace_utils::{
-                trace_move_call_end, trace_move_call_start, trace_ptb_summary, trace_split_coins,
-                trace_transfer,
+                add_move_value_info_from_obj_value, add_move_value_info_from_value,
```

Comment:
Not necessary for this PR (although feel free to do it in this one if you want) but I got thinking, just to make it more clear which functions are used for tracing only (e.g., with `add_move_value_infor_from_obj_value`) it would be nice to do an `trace_utils as TU,` and then qualify calls inside here through that (e.g., `TU::trace_execution_error`).